MapEditorReborn/Patches/UpdatePositionServerPatch.cs

```diff
@@ -16,7 +16,7 @@ namespace MapEditorReborn.Patches
     [HarmonyPatch(typeof(AdminToyBase), nameof(AdminToyBase.UpdatePositionServer))]
     internal static class UpdatePositionServerPatch
     {
-        private static bool Prefix(AdminToyBase __instance) => false;
```

Comment:
This affects all AdminToys, not only ones that are a part of CustomSchematic

Reply:
Even before I overidded this bit of code as it wasn't working as it should, also I safe a little bit performance not calling 3 properties per frame per object

Reply:
Then how i can use my plugin with MapEditorReborn? I cant spawn a lightsource/networkspawner::spawn()

https://github.com/Klarulor/SCPSL-TorchPlayer/blob/main/EventHandlerscs.cs#L23

Reply:
If you are using MER as a reference you can use MER methods to spawn stuff
`ObjectSpawner.SpawnLight()` should give you the same result

Reply:
Then i cant spawn without ur MER, right?

Reply:
you can, but it may be affected even if you don't want that

Reply:
if you don't need to use MER, just don't use it and use your own methods